SECURITY.md

```
@@ -1,5 +1,100 @@
+| 1.3.x     | :white_check_mark: |
+| 1.4.x     | :white_check_mark: |
+| 1.5.x     | :white_check_mark: |
+| 1.6.x     | :white_check_mark: |
+
+# Responsible Disclosure Policy
+
+Appwrite welcomes responsible security research and is committed to keeping our users, data, and infrastructure safe.
+
+## Contact
+
+We only accept **email** reports. Please use this address for all vulnerability reports: security@appwrite.io  
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)